src/containers/Topbar/Topbar.js

```diff
@@ -12,6 +12,23 @@ import { themeConfig } from '../../config';
 
 const FormItem = Form.Item;
 const { Header } = Layout;
+const addressMaxDisplayLength = 11;
```

Comment:
`ADDRESS_MAX_DISPLAY_LENGTH = 11;`? I think we should start standardizing the naming conventions.